parity-crypto/README.md

```diff
@@ -3,3 +3,8 @@
 General cryptographic utilities for Ethereum.
 
 By default, this library is compiled with the `secp256k1` feature, which provides ECDH and ECIES capability on that curve. It can be compiled without to avoid a dependency on the `libsecp256k1` library.
```

Comment:
(no introduced by this PR) do we actually expose that feature?

Reply:
We don't. Removing.